src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let countCalls = 0;
+
+  return (firstNumber) => {
```

Comment:
Consider renaming 'firstNumber' to 'a' for brevity and consistency with the task description.